### PR TITLE
change tapText style for better readability

### PR DESCRIPTION
**Pull request by tashiwangdi**
Making it bold makes it more readable.

### CODE REVIEW

File: lib/chart/animated_line_chart.dart [hunk 1]
```diff
@@ -242,7 +242,7 @@ class ChartPainter extends CustomPainter {
       TextSpan span = TextSpan(
           style: TextStyle(
               color: _chart.lines[index].color,
-              fontWeight: FontWeight.w200,
```
Comment (ghost) [inline lib/chart/animated_line_chart.dart:245]: Could we expose this field so it is configurable from users of the plugin?

Comment (tashiwangdi) [reply lib/chart/animated_line_chart.dart:245]: exposed the tap text fontweight

File: lib/chart/animated_line_chart.dart [hunk 2]
```diff
@@ -242,7 +242,7 @@ class ChartPainter extends CustomPainter {
...
+              fontWeight: tapTextFontWeight ?? FontWeight.w200,
```
Comment (ghost) [inline lib/chart/animated_line_chart.dart]: should w400 be the default?

**Review by ghost — APPROVED** [review]

File: lib/chart/animated_line_chart.dart (hunk 2 above)
Comment (tashiwangdi) [reply lib/chart/animated_line_chart.dart]: Done

**Pull request merged**